<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>码流分析工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>500</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>810</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>134</y>
        <width>981</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_unpack">
       <attribute name="title">
        <string>拆包管理</string>
       </attribute>
       <widget class="QTableView" name="tvw_unpack">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>654</width>
          <height>341</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QFrame" name="frame_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>5</y>
          <width>201</width>
          <height>80</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Stream Index:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cmb_index">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>3</y>
           <width>78</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Key Frame:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cmb_keyframe">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>78</width>
           <height>27</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>notKey</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>keyFrame</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QTextEdit" name="txt_unpackinfo">
        <property name="geometry">
         <rect>
          <x>656</x>
          <y>90</y>
          <width>321</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="frame_13">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>5</y>
          <width>772</width>
          <height>80</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>281</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>查询子句如：where pts=33 and flags=1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_where">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>50</y>
           <width>371</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>311</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>可用字段：stream_index,flags,pos,size,pts,dts</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_execsql">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>50</y>
           <width>71</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>EXEC</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>434</y>
          <width>976</width>
          <height>141</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTextEdit" name="txt_selectout">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>60</y>
           <width>971</width>
           <height>78</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>531</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>命令如：select sum(size) from avindex where stream_index =  0 and pts &lt; 9000</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_select">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>31</y>
           <width>551</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_execsql_2">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>30</y>
           <width>71</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>EXEC</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_analysis">
       <attribute name="title">
        <string>码流分析</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>116</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>981</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="txt_pkt_data">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>3</y>
         <width>971</width>
         <height>56</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>981</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>71</width>
         <height>49</height>
        </rect>
       </property>
       <property name="text">
        <string>File Path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>12</y>
         <width>511</width>
         <height>27</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_openfile">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>10</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>OPEN FILE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pause">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>10</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>PAUSE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stop">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>10</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
